src/odemis/driver/test/simcam_test.py

```diff
@@ -302,6 +310,83 @@ def test_focus(self):
         f.result()
         self.assertEqual(self.focus.position.value, pos)
 
+
+class TestSimCamWithPolarization(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.camera = CLASS(**KWARGS_POL)
```

Comment:
Does this test case work without the right backend running already? It seems to rely on  model.getComponent(). Please, drop this requirement. You should be able to create all the needed components (aka the pol-analyser and its children) in the setUpClass(). BTW, if you don't really want 4 components to have a fake pol-analyser, you could just use a simulated.Stage + FixedPositionActuator, which has 7 fixed positions associated to arbitrary positions of the stage axis.